Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -315,6 +325,12 @@ RDKit::SparseIntVect<boost::uint32_t> *MorganFingerprintHelper(
     python::object fromAtoms, bool useChirality, bool useBondTypes,
     bool useFeatures, bool useCounts, python::object bitInfo) {
   std::vector<boost::uint32_t> *invars = 0;
+  if (radius < 0) {
+        throw_value_error("radius " + NumberToString(radius) + " can not be below 0");;
+  }
+  if (nBits == 0) {
+        throw_value_error("nBits " + NumberToString(nBits) + " can not be 0");;
```

Comment:
Might be better to say something like, "nBits must be greater than 0",  NumberToString(nBits) will always be zero here so calling NumberToString is not needed.